test/train_voice.test.js

```diff
@@ -2027,20 +2027,20 @@ describe("Mama Nigeria App", function() {
             // TEST START ROUTING
 
             describe("Start of session", function() {
-                it("to state_set_language", function() {
+                it("to state_msg_receiver_msisdn", function() {
                     return tester
                         .setup.user.addr('+2345059991111')
                         .inputs(
                             {session_event: 'new'}
                             , '2'  // state_training_intro - change
                         )
                         .check.interaction({
-                            state: 'state_set_language'
+                            state: 'state_msg_receiver_msisdn'
                         })
                         .check.reply.properties({
                             helper_metadata: {
                                 voice: {
-                                    speech_url: ['http://localhost:8004/api/v1/eng_NG/state_set_language_1.mp3'],
+                                    speech_url: ['http://localhost:8004/api/v1/eng_NG/state_msg_receiver_msisdn_1.mp3'],
```

Comment:
Can you explain this test change? I'm sure it's right, I just don't understand how it relates to the code change.

Reply:
This is a voice line (rather than ussd) so we have sound files that play for each state. This test checks the first state that the user gets to, since we changed the first state we need to make sure that the correct sound file plays.